src/config.rs

```diff
@@ -167,24 +183,37 @@ pub struct ServerConfig {
 
     /// TLS configuration for helper-to-helper communication
     pub tls: Option<TlsConfig>,
+
+    /// Configuration needed for encrypting and decrypting match keys
+    pub matchkey_encryption_info: Option<MatchKeyEncryptionConfig>,
```

Comment:
@andyleiserson I have stored public and private keys as optional as MatchKeyEncryptionConfig in ServerConfig. Lets leave it optional for now to support disabling encryption of match keys

Reply:
Added a flag in net/test.rs for disable_matchkey_encryption but not adding any test for that as yet. Will add it along with coding decryption handling of the reports